extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -662,6 +662,11 @@ export class AmpA4A extends AMP.BaseElement {
               })
               .then(status => {
                 this.protectedEmitLifecycleEvent_('adResponseValidateEnd');
+                if (getMode().localDev &&
```

Comment:
@taymonbeal I tried compiled with `gulp dist`. The clock is being excluded even not nested.